src/coreclr/System.Private.CoreLib/src/System/GC.cs

```diff
@@ -423,6 +424,8 @@ public static GCNotificationStatus WaitForFullGCApproach(int millisecondsTimeout
             return (GCNotificationStatus)_WaitForFullGCApproach(millisecondsTimeout);
         }
 
+        public static GCNotificationStatus WaitForFullGCApproach(TimeSpan timeout) => WaitForFullGCApproach(WaitHandle.ToTimeoutMilliseconds(timeout));
```

Comment:
Can you please add doc comments for these new public members? I know the others are missing them, but we're progressively moving to generating docs from sources, it's just we don't do it here yet.

Reply:
oh perhaps you can just copy them all from the other GC.cs.